src/TagLib/Ogg/XiphComment.cs

```diff
@@ -1122,18 +1128,32 @@ public override string[] PerformersRole
 		///    current instance, or zero if not specified.
 		/// </value>
 		/// <remarks>
-		///    This property is implemented using the "TEMPO" field.
+		///    This property is implemented using the "TEMPO" field.
+		///    Since there is no official definition, this property is
+		///    also implemented using the "BPM" field.
 		/// </remarks>
 		public override uint BeatsPerMinute {
-			get {
+			get {
+				save_BPM_as_tempo = true;
 				string text = GetFirstField ("TEMPO");
+				if (string.IsNullOrEmpty(text))
+				{
+					text = GetFirstField("BPM");
+					if (!string.IsNullOrEmpty(text))
+					{
+						save_BPM_as_tempo = false;
+					}
+				}
 				double value;
 				return (text != null &&
-					double.TryParse (text, out value) &&
+					double.TryParse (text, System.Globalization.NumberStyles.AllowDecimalPoint, System.Globalization.NumberFormatInfo.InvariantInfo, out value) &&
```

Comment:
I wouldn't put the fully qualified namespace in here. we already have a `using System.Globalization` at the top. Also it pushes the code off the screen because of the length.

Reply:
Updated